ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -824,6 +824,7 @@ Wait For Model KServe Deployment To Be Ready
     ...                the deployment has the expected pods and containers
     [Arguments]    ${label_selector}    ${namespace}    ${runtime}
     ...    ${timeout}=300s    ${exp_replicas}=${1}
+    Wait For Pods To Be Ready    label_selector=${label_selector}    namespace=${namespace}
```

Comment:
If we don't specify a timeout here it's going to wait for 10 minutes, is that ok? 
IMHO yes so approving :D

Reply:
![image](https://github.com/red-hat-data-services/ods-ci/assets/31654558/51b45fcd-bbbb-4171-8691-19be79bdc000)
The wait is active (not waiting 10 minutes and then checking)
So it's ok for me too :)